docs/02-guides/customize-wysiwyg-platform.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_position: 4
+hide_table_of_contents: false
+title: Customize WYSIWYG Platform
+description:
+  In this guide, you will learn the different WYSIWYG implementations available
+  on Front-Commerce and how to customize them.
+---
+
+import TOCInline from "@theme/TOCInline";
+import ContactLink from "@site/src/components/ContactLink";
+import SinceVersion from "@site/src/components/SinceVersion";
```

Comment:
No need to import these, they are added to the component library: https://github.com/front-commerce/developers.front-commerce.com/blob/main/src/theme/MDXComponents.ts